trl/trainer/dpo_trainer.py

```diff
@@ -1052,121 +933,116 @@ def null_ref_context(self):
             if self.ref_adapter_name:
                 self.model.set_adapter(self.model_adapter_name or "default")
 
-    def compute_reference_log_probs(self, padded_batch: Dict) -> Dict:
+    def compute_reference_log_probs(self, batch: Dict[str, torch.LongTensor]) -> Dict:
```

Comment:
turbo nit: since we're already renaming variables, maybe we can align the method too?
```suggestion
    def compute_ref_log_probs(self, batch: Dict[str, torch.LongTensor]) -> Dict:
```

Reply:
done in e509549